stdlib/ballerina-builtin/src/main/ballerina/ballerina/util/natives.bal

```diff
@@ -31,3 +33,13 @@ public native function base16ToBase64Encode (string baseString) (string);
 @Param {value:"s: string to be encoded"}
 @Return {value:"the encoded string."}
 public native function base64ToBase16Encode (string baseString) (string);
+
+@Description {value:"Convert hex Numbers into decimal Numbers"}
+@Param {value:"hexValue: string of the hex number to be converted into decimal"}
+@Return {value:"integer in decimal"}
+public native function hexToDecimal (string hexValue) (int, TypeConversionError);
+
+@Description {value:"Convert decimal Numbers into hex Numbers"}
+@Param {value:"decimalValue: integer of the decimal number to beconverted into hex"}
+@Return {value:"string of hex number"}
+public native function decimalToHex (int decimalValue) (string);
```

Comment:
Typos in the doc comments. 
- Capitalize the first letter in param and return val descriptions
- Numbers -> numbers
- Missing space in "beconverted", `@Param` annotation of `decimalToHex()`